School/utils/add_schools_to_db.go

```diff
@@ -37,12 +35,11 @@ func addSchoolsToDB() {
 	defer tx.Rollback(context.Background())
 
 	var schools []domain.School
-	resp, _ := http.Get("https://raw.githubusercontent.com/Hipo/university-domains-list/master/world_universities_and_domains.json")
-	schoolJSON, _ := ioutil.ReadAll(resp.Body)
+	jsonFile, _ := os.Open("school.json")
```

Comment:
should this change be in this PR? It looks like you are also doing the stud-191 bug fix, can you please mention the 2 issue id's in the description of this PR